README.md

```diff
@@ -854,7 +854,96 @@ Operators: []definition.Operator{
 
 ### OpenAPI
 
-TODO
+Nirvana can generate OpenAPI 2.0 document from code simply.
+
+In the example, swagger will be generated by builder of OpenAPI spec. 
```

Comment:
we already have example above, i think this should be a thorough guide.  what it is, how it works, etc

Reply:
?!!!

Reply:
What I need to explain? How to generate code?

Reply:
let's discuss this offline tmr